app/javascript/oldjs/controllers/timeline/timeline_options_controller.js

```diff
@@ -31,6 +33,14 @@ ManageIQ.angular.app.controller('timelineOptionsController', ['miqService', 'url
     vm.reportModel.tl_categories = [];
   };
 
+  vm.showFilterField = function() {
+    let bool = false;
+    if(vm.model === "PhysicalStorage" || vm.model === "ExtManagementSystem"){
```

Comment:
@Oferlis The `vm.model === "ExtManagementSystem"` shouldn't be used because if you go into the monitoring page for a cloud provider it will have the value of `"ExtManagementSystem"` for `vm.model` and cause this field to be rendered there.

<img width="1470" alt="Screen Shot 2022-09-06 at 6 46 54 PM" src="https://user-images.githubusercontent.com/32444791/188754062-4f98b587-5e1c-48b0-bf81-03487b8f1d31.png">

Reply:
Just changed it to rely on the Autosde model so any other model will not be effected.
Changes will be pushed today.